compiler/src/iree/compiler/Codegen/LLVMCPU/test/select_x86_64_lowering_strategy.mlir

```diff
@@ -1648,7 +1648,7 @@ hal.executable private @pack_many_elements  {
     }
   }
 }
-//  CHECK-DAG: #[[CONFIG:.+]] = #iree_codegen.lowering_config<tile_sizes = {{\[}}[100, 31250], [1, 1]]>
+//  CHECK-DAG: #[[CONFIG:.+]] = #iree_codegen.lowering_config<tile_sizes = {{\[}}[50, 48], [1, 1]]>
```

Comment:
This is the case that I'm talking about in the other comment. This is just a memcpy op, but we only move `16*50*48*f32 = 153600 ` bytes (i.e., 150KB) in one kernel launch. We probably want to make tile sizes larger, what do you think?